python/utils/worker.py

```diff
@@ -488,7 +517,23 @@ def run(self):
                 self.processing_result = self.processor.process_transactions(
                     self.transactions, start_version, end_version
                 )
-
+                logging.info(
```

Comment:
Each metric should have its own log statement (like how it's logged in Rust). This makes it easier to plot all in the same chart in Axiom.

Let's split out into 3 log statements for each (db insertion, parsing duration, overall processing duration) and use the `duration_in_secs` extra field.

Reply:
done.